tests/components/moehlenhoff_alpha2/__init__.py

```diff
@@ -1 +1,38 @@
 """Tests for the moehlenhoff_alpha2 integration."""
+
+from unittest.mock import patch
+
+import xmltodict
+
+from homeassistant.components.moehlenhoff_alpha2.const import DOMAIN
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry, load_fixture
+
+MOCK_BASE_HOST = "fake-base-host"
+
+
+async def mock_update_data(self):
+    """Mock moehlenhoff_alpha2.Alpha2Base.update_data."""
+    data = xmltodict.parse(load_fixture("static2.xml", DOMAIN))
+    for _type in ("HEATAREA", "HEATCTRL", "IODEVICE"):
+        if not isinstance(data["Devices"]["Device"][_type], list):
+            data["Devices"]["Device"][_type] = [data["Devices"]["Device"][_type]]
+    self.static_data = data
+
+
+async def init_integration(hass: HomeAssistant) -> MockConfigEntry:
+    """Mock integration setup."""
+    with patch("moehlenhoff_alpha2.Alpha2Base.update_data", mock_update_data):
```

Comment:
Can we patch this at the place where its used?

Reply:
No idea... I just moved/reused the existing patch

Reply:
Updated